src/project-manager.ts

```diff
@@ -382,9 +388,10 @@ class InMemoryLanguageServiceHost implements ts.LanguageServiceHost {
         this.versions = versions;
         this.projectVersion = 1;
         this.files = [];
-        // adding content of default library file read from the local file system 
-        const defaultLib = util.normalizePath(this.getDefaultLibFileName(options));
-        this.fs.entries[defaultLib] = fs_.readFileSync(defaultLib).toString();
+        // adding library files from the local file system
+        readTsLibraries().forEach((content, name) => {
```

Comment:
We are fetching content of libraries TS compiler carries with itself. See "lib" here: http://json.schemastore.org/tsconfig. We do not read content of local file system outside our `node_modules/typescript/*.d.ts`